mixer/pkg/server/server.go

```diff
@@ -66,7 +66,7 @@ type listenFunc func(network string, address string) (net.Listener, error)
 // replaceable set of functions for fault injection
 type patchTable struct {
 	newRuntime func(s store.Store, templates map[string]*template.Info, adapters map[string]*adapter.Info,
-		identityAttribute string, defaultConfigNamespace string, executorPool *pool.GoroutinePool,
+		defaultConfigNamespace string, executorPool *pool.GoroutinePool,
```

Comment:
I think you also need to remove the identityAttribute command-line option to Mixer, and the associated parameter in the ServerArgs struct

Reply:
Done. Surprising amount of use of this "hidden" flag throughout the code base...